core/src/apps/management/recovery_device/word_validity.py

```diff
@@ -0,0 +1,105 @@
+import storage.recovery
+from trezor.messages import BackupType
+
+from apps.management.recovery_device import recover
+
+if False:
+    from typing import List, Optional
+    from trezor.messages.ResetDevice import EnumTypeBackupType
+
+
+class WordValidityResult(BaseException):
```

Comment:
this should be `Exception`, unless there's a strong reason for `BaseException`?

Reply:
Fixed.